### ISSUE TITLE
config: Log_Level setting does not take env variable

### ISSUE THREAD

**Issue by nauckunas** [fluent/fluent-bit#920]
## Bug Report

**Describe the bug**
fluentbit does not respect `Log_Level` setting when set via environment variable. It works if statically set in configuration file.

Example:
```
[SERVICE]
    Flush        5
    Daemon       Off
    Log_Level ${FLB_LOG_LEVEL}
```
```
$ env | grep FLB_LOG_LEVEL
FLB_LOG_LEVEL=debug
```

This results to some default log level. I think that is `info`.
Tested using below steps.
```
$ docker run --log-driver=fluentd -t alpine echo "Testing a log message"
Testing a log message
```
Fluentbit logs:
```
Fluent-Bit v0.14.7
Copyright (C) Treasure Data

[2018/11/23 13:21:43] [ info] [engine] started (pid=1)
[2018/11/23 13:21:43] [ info] [in_fw] binding 0.0.0.0:24224
```

**To Reproduce**

1. Change fluentbit configuration to set Log_Level via environment variable
2. Set environment variable
3. Restart fluentbit with new configuration (make sure that env var is set for the process)

**Expected behavior**
Log_Level set based on environment variable

**Your Environment**
* Version used: 0.14.7
* Configuration: 
```
[SERVICE]
    Flush        5
    Daemon       Off
    Log_Level ${FLB_LOG_LEVEL}
[INPUT]
    Name Forward
    Listen 0.0.0.0
    Port 24224
[OUTPUT]
    Name  es
    Match *
    Host  ${ES_HOST}
    Port  443
    Logstash_Format On
    Logstash_Prefix ${INDEX_PREFIX}
    Logstash_DateFormat %Y.%m.%d
    Retry_Limit     5
    TLS             On
```

**Additional context**
This would allow to change fluentbit log level if required for debug other other purposes via environment variable. Without requirement to change configuration file.

**Comment by edsiper** [comment]
thanks for reporting this problem. It has been fixed by a27991b8.

The fix will be available on 0.14.8 release today.

**Comment by saviogl** [comment]
@edsiper Is that a chance that it's also happening for http **output** ?:

```
    [OUTPUT]
        Name  http
        Match *
        Host  ${HTTP_HOST}
        Port  ${HTTP_PORT}
        URI   ${HTTP_URI}
        Format json_lines
        tls        On
        tls.verify Off
```
